### ISSUE TITLE
[iam-role-for-service-accounts-eks] cert-manager policy

### ISSUE THREAD

**Issue by dmitriishaburov** [terraform-aws-modules/terraform-aws-iam#217]
## Is your request related to a new offering from AWS?
No

## Is your request related to a problem? Please describe.
Support for IAM role for cert-manager https://cert-manager.io/docs/configuration/acme/dns01/route53/#set-up-an-iam-role

## Describe the solution you'd like.
I can provide a pull request, based on external-dns policy with minimal changes

## Describe alternatives you've considered.
Making custom resources

## Additional context
Basically just creating issue to check if it's OK to provide pull request for cert-manager support

**Comment by antonbabenko** [comment]
This issue has been resolved in version 4.20.0 :tada:
